### ISSUE TITLE
False positive: RequireMultiLineTernaryOperatorSniff takes comments on lines before into account

### ISSUE THREAD

**Issue by Levivb** [slevomat/coding-standard#545]
Hey,

It seems the RequireMultiLineTernaryOperatorSniff takes the comments on the line prior to a ternary statement into account.
Code snippet with line length 120 set:
```php
        // this ternary statement is reported as to long
        // Determine the class name of class for which we are calling the 'extenders'
        $classNameToExtend = is_string($classObject) ? $classObject : get_class($classObject);
```

line length is reported as 237 characters while the actual line is 94 characters.

The following is also reported as to long:
```php
        $this->feed->setDateFormat('datetime'); // 'datetime', 'timestamp' or 'carbon'
        $this->feed->pubdate = $this->items->first() ? $this->items->first()->created_at : date('Y-m-d H:i:s');
```
As it counts the ` // 'datetime', 'timestamp' or 'carbon'` part from previous line as well

Thanks for looking into it!

Regards

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/1caf2fafad1b7399f2c7acf4f21890e629180bca
